WindowServer+LibGUI+LibGfx+LibVT+Base: Add new DragCopy Cursor

krkk opened the pull request:
https://user-images.githubusercontent.com/16520278/185796449-89d97947-7550-433e-a67d-2a565a879be9.mp4

for now only TerminalWidget, Breadcrumbbar, and any (editable) view model use this cursor.

~~also, as you can see, the drag overlay jumps a bit, cuz DragCopy cursor and the normal Drag cursor have different sizes :/~~ Fixed!!

AtkinsSJ commented:
Cool feature! For the cursor jumping issue, could you increase the grab cursor's size to match the grab-drop one? It's a bit of a hack but I don't know if there's a better solution right now.

krkk commented:
Changes since last push:
- rebased to master

- fixed overlay jumps (by using the current cursor position instead of center of a cursor bitmap) (https://github.com/SerenityOS/serenity/pull/14957/commits/8d79d784ab4f4c9c9a1377afd1e3b9a2fb048b1b):

  ![preview](https://user-images.githubusercontent.com/16520278/187186033-6a249b04-8eaa-44ed-a257-42e6612ee270.gif)

- added 2x DragCopy cursors (and missing 2x Drag cursor for Dark theme (https://github.com/SerenityOS/serenity/pull/14957/commits/9799e7968a9bd96ddf8eec6ecf39f2fd79815297))
  
  > Failing to load 2x cursor resulted in using the default arrow cursor instead of scaling the 1x one.
  
  ![:yakbait:](https://cdn.discordapp.com/emojis/873672505309679758.png?size=20)

AtkinsSJ commented:
Very nice! :^)
